src/pages/api/broker/account/balance/index.ts

```diff
@@ -0,0 +1,36 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { ResponseDataType } from "../../../../../db/ResponseDataMeta";
+import { getAccountCashBalance } from "../../../../../services/alpacaService";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse,
+) {
+  try {
+    const responseData: ResponseDataType = { status: "INIT" };
+    await getAccountCashBalance()
+      .then((result) => {
+        responseData.status = "OK";
+        responseData.meta = {
+          balance: result,
```

Comment:
I know I introduced this kind of response structure, but it looks more accurate if the response structure has the main intended data "higher up" than within "meta". Like:
`{
   status: "OK",
   balance: result,
   meta: {...misc other details that might be interesting, but not crucial - and if we dont want to pass misc additional information here, just skip returning the "meta"-prop}
}`